bindings/ergo-lib-wasm/src/transaction/reduced.rs

```diff
@@ -68,6 +81,99 @@ impl ReducedTransaction {
     pub fn unsigned_tx(&self) -> UnsignedTransaction {
         self.0.unsigned_tx.clone().into()
     }
+
+    /// Getting first input box serialized sigma prop bytes
+    pub fn get_first_input_serialized_bytes(&self) -> Result<Vec<u8>, JsValue> {
+        self.0
+            .reduced_inputs()
+            .first()
+            .clone()
+            .reduction_result
+            .sigma_prop
+            .sigma_serialize_bytes()
+            .map_err(to_js)
+    }
+
+    /// Generate commitment for first input with input address
+    pub fn generate_commitment_for_first_input(
```

Comment:
Why do we need it? I don't see it is used anywhere in the code? The same question also goes to `get_first_input_serialized`

Reply:
Now should be fixed.

Reply:
Thanks!